packages/kit/test/utils.js

```diff
@@ -169,7 +169,8 @@ export const config = {
 	use: {
 		screenshot: 'only-on-failure',
 		trace: 'retain-on-failure'
-	}
+	},
+	workers: process.env.CI ? 2 : undefined
```

Comment:
we had removed this purposefully not long ago. a lot of the CIs have only 1 CPU and we were worried that having multiple workers was making them compete for CPU and causing timeouts

Reply:
This is a test, but adding full parallelization to the tests (at the pnpm level) didn't cause contention to the point of increasing run time, in fact it reduced a few percent. This is also a "max workers" setting—shouldn't playwright still use 1 worker when there's only one cpu?

Reply:
I don't think adding more workers would increase overall runtime, but it may make individual tests take longer to run

But anyway it looks like I may have been wrong that some environments only support a single CPU? I think I'm seeing now that most support 2, so this would be fine